lib/src/cupertino/cupertino_controls.dart

```diff
@@ -754,6 +757,10 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position - const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.max(skip, beginning)));
+    log("Reversing the video", name: "Video Player");
```

Comment:
Please remove all logging. Thanks.

Reply:
All the logging has been removed now. 
Thanks for the feedback.